repos/fdbt-site/src/pages/service.tsx

```diff
@@ -108,9 +108,13 @@ export const getServerSideProps = async (ctx: NextPageContextWithSession): Promi
 
     const dataSourceAttribute = getRequiredSessionAttribute(ctx.req, TXC_SOURCE_ATTRIBUTE);
 
-    const services = await getServicesByNocCodeAndDataSource(nocCode, dataSourceAttribute.source);
+    const services: ServiceType[] = await getServicesByNocCodeAndDataSource(nocCode, dataSourceAttribute.source);
+    const unqiueServiceIds = getMyUniqueServices(services);
+    const servicesWithNoDuplicates = services.filter(
```

Comment:
Is there a reason why we cant move this filter into the function which gets unique services? Why do we have to do it here?